<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>823</width>
    <height>674</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>The MP3 Decider</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0" rowspan="2">
       <widget class="QGroupBox" name="groupBox">
        <property name="title">
         <string>Current Track Info</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QLabel" name="m_trackTitleLabel">
           <property name="font">
            <font>
             <pointsize>20</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Track Title</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="m_trackArtistLabel">
           <property name="font">
            <font>
             <pointsize>16</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Artist</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="m_trackAlbumLabel">
           <property name="font">
            <font>
             <pointsize>16</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Album</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="m_trackFilenameLabel">
           <property name="font">
            <font>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Filename</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QLCDNumber" name="m_trackTime">
             <property name="minimumSize">
              <size>
               <width>100</width>
               <height>0</height>
              </size>
             </property>
             <property name="smallDecimalPoint">
              <bool>false</bool>
             </property>
             <property name="segmentStyle">
              <enum>QLCDNumber::Flat</enum>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_5">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item row="1" column="1">
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>18</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QToolButton" name="m_restartToolButton">
        <property name="minimumSize">
         <size>
          <width>128</width>
          <height>128</height>
         </size>
        </property>
        <property name="text">
         <string>...</string>
        </property>
        <property name="icon">
         <iconset resource="res/MP3Decider.qrc">
          <normaloff>:/MainWindow/ReplayButton.png</normaloff>:/MainWindow/ReplayButton.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>128</width>
          <height>128</height>
         </size>
        </property>
        <property name="autoRaise">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="m_rewindToolButton">
        <property name="minimumSize">
         <size>
          <width>128</width>
          <height>128</height>
         </size>
        </property>
        <property name="text">
         <string>...</string>
        </property>
        <property name="icon">
         <iconset resource="res/MP3Decider.qrc">
          <normaloff>:/MainWindow/RewindButton.png</normaloff>:/MainWindow/RewindButton.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>128</width>
          <height>128</height>
         </size>
        </property>
        <property name="autoRepeat">
         <bool>true</bool>
        </property>
        <property name="autoRaise">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="m_playToolButton">
        <property name="minimumSize">
         <size>
          <width>128</width>
          <height>128</height>
         </size>
        </property>
        <property name="text">
         <string>...</string>
        </property>
        <property name="icon">
         <iconset resource="res/MP3Decider.qrc">
          <normaloff>:/MainWindow/PlayButton.png</normaloff>:/MainWindow/PlayButton.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>128</width>
          <height>128</height>
         </size>
        </property>
        <property name="autoRaise">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="m_fastForwardToolButton">
        <property name="minimumSize">
         <size>
          <width>128</width>
          <height>128</height>
         </size>
        </property>
        <property name="text">
         <string>...</string>
        </property>
        <property name="icon">
         <iconset resource="res/MP3Decider.qrc">
          <normaloff>:/MainWindow/FastForwardButton.png</normaloff>:/MainWindow/FastForwardButton.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>128</width>
          <height>128</height>
         </size>
        </property>
        <property name="autoRepeat">
         <bool>true</bool>
        </property>
        <property name="autoRaise">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="m_stopToolButton">
        <property name="minimumSize">
         <size>
          <width>128</width>
          <height>128</height>
         </size>
        </property>
        <property name="text">
         <string>...</string>
        </property>
        <property name="icon">
         <iconset resource="res/MP3Decider.qrc">
          <normaloff>:/MainWindow/StopButton.png</normaloff>:/MainWindow/StopButton.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>128</width>
          <height>128</height>
         </size>
        </property>
        <property name="autoRaise">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_4">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QToolButton" name="m_badTrackToolButton">
        <property name="minimumSize">
         <size>
          <width>128</width>
          <height>128</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Copy track to bad folder</string>
        </property>
        <property name="text">
         <string>...</string>
        </property>
        <property name="icon">
         <iconset resource="res/MP3Decider.qrc">
          <normaloff>:/MainWindow/BadFileButton.png</normaloff>:/MainWindow/BadFileButton.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>128</width>
          <height>128</height>
         </size>
        </property>
        <property name="autoRaise">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="m_skipTrackToolButton">
        <property name="minimumSize">
         <size>
          <width>128</width>
          <height>128</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Skip track and leave track in source folder</string>
        </property>
        <property name="text">
         <string>...</string>
        </property>
        <property name="icon">
         <iconset resource="res/MP3Decider.qrc">
          <normaloff>:/MainWindow/DoNotMoveButton.png</normaloff>:/MainWindow/DoNotMoveButton.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>128</width>
          <height>128</height>
         </size>
        </property>
        <property name="autoRaise">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="m_goodTrackToolButton">
        <property name="minimumSize">
         <size>
          <width>128</width>
          <height>128</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Copy track to good folder</string>
        </property>
        <property name="text">
         <string>...</string>
        </property>
        <property name="icon">
         <iconset resource="res/MP3Decider.qrc">
          <normaloff>:/MainWindow/GoodFileButton.png</normaloff>:/MainWindow/GoodFileButton.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>128</width>
          <height>128</height>
         </size>
        </property>
        <property name="autoRaise">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QPlainTextEdit" name="m_logTextEdit"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>823</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuConfigure">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionConfigure"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <addaction name="menuConfigure"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionExit">
   <property name="text">
    <string>E&amp;xit</string>
   </property>
   <property name="toolTip">
    <string>Exit the Decider</string>
   </property>
  </action>
  <action name="actionConfigure">
   <property name="text">
    <string>&amp;Configure</string>
   </property>
   <property name="toolTip">
    <string>Configure the Decider</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="res/MP3Decider.qrc"/>
 </resources>
 <connections/>
</ui>
